src/components/container/ConnectDevicesMenu/ConnectDevicesMenu.tsx

```diff
@@ -65,18 +70,22 @@ function useConnectDevicesMenuState(
             deviceExternalAccounts: [],
             platform: "Web",
             hasRecentAppleHealthData: false,
-            healthConnectStatus: null
+            healthConnectStatus: null,
+            participantInfo: sampleParticipantInfo
         }
 
         if (previewState == "ConnectedStates") {
             previewStateObject.deviceExternalAccounts = previewAccounts;
             previewStateObject.platform = "iOS";
             previewStateObject.hasRecentAppleHealthData = true;
+            previewStateObject.participantInfo!.demographics.postalCode = "00000";
         }
         else if (previewState == "iOS") {
             previewStateObject.platform = "iOS";
         } else if (previewState == "Android") {
             previewStateObject.platform = "Android";
+        } else {
+            previewStateObject.participantInfo!.demographics.postalCode = "";
```

Comment:
I'm not sure this is necessary.  The `sampleParticipantInfo` looks like it will already have an empty string postal code.

Reply:
Annoyingly, it is, because of what I'm doing on line 74.  That change ends up persisting when you move between stories if you don't reset it here.  Changing line 74 to `participantInfo: {...sampleParticipantInfo}` isn't sufficient either because both things end up with a reference to the same `demographics` object.  I could avoid this by doing a deep copy on line 74 with JSON.parse tricks, but that seemed almost excessive.

Reply:
Ah, I see what you mean.

Not a big deal, but if you want to drop this, you could probably export a `createSampleParticipantInfo()` function instead of exporting the `sampleParticipantInfo` object directly.  Then each story would have a different one.